adobe/react-spectrum
# Fix SearchWithin aria-label and aria-labelledby

Closes #2174

## ✅ Pull Request Checklist:


## 📝 Test Instructions:

- Check id, aria-label, and aria-labelledby of group, input, and picker on 'Default', 'No Visible Label', and 'No Label' stories. 
- Check SearchWithin tests that test labels. Verify that it matches desired functionality mentioned in #2174.

Label combinations should look something like this:
```
<SearchWithin label=foo
searchwithin group is labelledby the label
input is labelled by searchwithin group
picker is labelled by searchwithin group + itself + dropdown label

<SearchWithin aria-label=foo
searchwithin group has aria-label foo
input is labelled by searchwithin group
picker is labelled by searchwithin group + itself + dropdown label

<SearchWithin
searchwithin group has aria-label w/default value of  "Search"
input is labelled by searchwithin group
picker is labelled by searchwithin group + itself + dropdown label
```

## 🧢 Your Project:

RSP
